### DISCUSSION TITLE
HA integration stopped working on 23 Nov 23

### DISCUSSION THREAD

**Discussion by mitasa** [flobz/psa_car_controller#680] [Q&A]
Looks like something happened with the PSA API yesterday, the HA integration does not get any updates on the sensors. HA RESTful get error 

REST result could not be parsed as JSON

and PSA Car Controller log shows 

2023-11-24 14:22:22,064 :: ERROR :: get_vehicle_info: ApiException: (400) Reason: Bad Request

HTTP response body: {"uuid":"42424b60-9a48-4895-aed2-b62a6964ab23","code":40000,"message":" Invalid parameter: X-MPHSource","timestamp":"2023-11-24T12:22:21.979Z"}

023-11-24 14:22:22,068 :: ERROR :: status of VX.....XXX is None
2023-11-24 14:22:22,069 :: ERROR :: Probably can't retrieve all information from API:

**Comment by Apollon77** [comment]
duplicate #678 
duplicate #674 
fixedby #679
